src/renderer/viz/expressions/scale.js

```diff
@@ -0,0 +1,48 @@
+import BaseExpression from './base';
```

Comment:
The expression class LGTM ✨. I would add:

1. specs
2. rename the file to `Scale.js`.